src/rust_version.rs

```diff
@@ -37,3 +53,16 @@ fn fetch_rust_version() -> Option<String> {
     *CACHE.write().unwrap() = Some((version.clone(), Instant::now()));
     Some(version)
 }
+
+fn fetch_rust_release_post() -> Option<String> {
+    let releases = reqwest::get("https://blog.rust-lang.org/releases.json")
+        .ok()?
+        .text()
+        .ok()?;
+    let releases = releases.parse::<serde_json::Value>().ok()?;
+    let url = releases["releases"][0]["url"].as_str()?.to_string();
```

Comment:
Just realized. Should this be `releases["releases"][0]`?

Reply:
Not sure I follow you - and local testing indicated that the site works and correctly links.

Reply:
It works because there is only one release right now. But I'm uncertain the order if there are multiple releases. Should it be the first element or the last?

Reply:
Ah. Hm. I believe it'll be the first post if I understand the blog code correctly so we should be fine; if it doesn't work then changing it is fairly easy so I'm not too worried.